### DISCUSSION TITLE
I am not getting useful output after running log4j security scanner

### DISCUSSION THREAD

**Discussion by teo-en-ming** [adilsoybali/Log4j-RCE-Scanner#8] [Q&A] [answered]
Hi!

I am not getting useful output after running your log4j security scanner.

The scan result I am getting is only:

Subfinder is working 
 Assetfinder is working 
 Amass is working

There is no other output.

Should I be getting more output?

Please help me troubleshoot.

I have documented my attempts in 3 blog posts. Please have a look at them.

[ERROR ENCOUNTERED] 1st attempt at installing and running log4j/log4shell remote command execution security vulnerability scanner developed by Adil Soybali

Link: https://tdtemcerts.blogspot.com/2021/12/error-encountered-1st-attempt-at.html

[ERROR ENCOUNTERED] 2nd attempt at installing and running log4j/log4shell remote command execution security vulnerability scanner developed by Adil Soybali

Link: https://tdtemcerts.blogspot.com/2021/12/error-encountered-2nd-attempt-at.html

Success on the 3rd attempt in getting log4j/log4shell remote command execution security vulnerability scanner developed by Adil Soybali to run without HTTPX error message

Link: https://tdtemcerts.blogspot.com/2021/12/success-on-3rd-attempt-in-getting.html

I am looking forward to your reply.

Thank you very much.

Mr. Turritopsis Dohrnii Teo En Ming
Targeted Individual in Singapore
18 Dec 2021 Saturday

**Comment by adilsoybali** [comment, marked as answer]
Hello @teo-en-ming. Sorry for the delay. The tool does not work, it gives an error from the httpx tool because you installed the wrong httpx tool. You have installed the httpx tool, which is Python, but this tool works with the httpx tool written with go. Please uninstall httpx tool written in python and install httpx tool written in go.

Python httpx tool: https://www.python-httpx.org/
The httpx tool we use, written in Go: https://github.com/projectdiscovery/httpx

Had the same problem with this: https://github.com/adilsoybali/Log4j-RCE-Scanner/issues/7

**Comment by teo-en-ming** [comment]
Hi @adilsoybali thank you for your reply.

I have already solved the httpx issue/problem before I posted the question here. My solution for the httpx problem is documented on my blog post.

Blog post: Success on the 3rd attempt in getting log4j/log4shell remote command execution security vulnerability scanner developed by Adil Soybali to run without HTTPX error message

Link: https://tdtemcerts.blogspot.com/2021/12/success-on-3rd-attempt-in-getting.html

However, when I run your log4j tester, it keeps resetting the virtual network adapter in my Ubuntu 21.10 Linux virtual machine.

Please take a look at the problem. Thank you.

Blog post: Running adilsoybali log4j tester resets virtual network adapter in Ubuntu 21.10 Linux virtual machine

Link: https://tdtemcerts.blogspot.com/2021/12/running-adilsoybali-log4j-tester-resets.html

Looking forward to your reply. Thank you.

Mr. Turritopsis Dohrnii Teo En Ming
Targeted Individual in Singapore
19 Dec 2021 Sunday

**Reply by adilsoybali** [reply]
Hello @teo-en-ming. This tool does not do any network settings. So I think this problem is caused by something else.

In addition, I request you to remove the following content in order not to cause information pollution.
1. https://lore.kernel.org/all/U-14mS6_LpnbfmKXe3Y6bgDsk_l-ET44qYU79c9hWGjEet5lahAbRkfHk2pTAU8JWdi5u0EE0kBH82MQymzI5G34jmnIC3I1b4eBqG_4u1s=@protonmail.com/ 
2. https://tdtemcerts.blogspot.com/2021/12/running-adilsoybali-log4j-tester-resets.html

Thanks
